bulbs/api/views.py

```diff
@@ -450,8 +450,8 @@ def list(self, request):
                 results.append(dict(
                     name=name,
                     doctype=doctype
-                ))
-        results.sort(key=lambda x: x["name"])
+                    ))
+                results.sort(key=lambda x: x["name"])
```

Comment:
I don't know what happened here.  I definitely don't recall going through this file and changing indentations ???

But it's definitely me from the blame.

Maybe this explains my fever code dreams ;)

Will look through the diff on this file and revert them.